tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -1091,6 +1092,12 @@ gnmi/test_gnmi_configdb.py:
       - "'t2' in topo_name"
       - "is_multi_asic==True"
 
+gnmi/test_gnoi_killprocess.py::test_gnoi_killprocess_then_restart[dhcp_relay-True-]:
+  skip:
+    reason: "DHCP relay is not enabled on BT0"
+    conditions:
+      - "'t0-isolated' in topo_name"
```

Comment:
should we ignore this test by checking feature status? many dhcp tests is doing it, do you mind to help check it?